tensorflow_estimator/python/estimator/run_config.py

```diff
@@ -311,8 +311,9 @@ def _validate_delay(delay):
                     ' one argument "op".')
 
   _validate('protocol',
-            lambda protocol: protocol in (None, "grpc", "grpc+verbs"),
-            message='protocol should be grpc or grpc+verbs')
+            lambda protocol: protocol in (
+                None, "grpc", "grpc+verbs", "grpc+gdr", "grpc+mpi", "grpc+seastar"),
+            message='protocol should be either grpc, grpc+verbs, grpc+gdr, grpc+mpi, or grpc+seastar')
```

Comment:
Do we have to do anything to actually ensure that these work? What does it mean to pass these through without tests or actual code to handle the changes?

Reply:
Right now we do not have protocol registration mechanism in Python. I will submit a separate PR in TF core to expose available protocol information. Ping @annarev to confirm.